.github/workflows/run_tests.yml

```diff
@@ -24,134 +24,59 @@ jobs:
-            blueprint_algebra_fields_plonk_non_native_bit_decomposition_test,
-            blueprint_algebra_fields_plonk_non_native_bit_composition_test,
-            blueprint_algebra_fields_plonk_non_native_bit_shift_constant_test,
-            blueprint_algebra_fields_plonk_non_native_logic_ops_test,
-            blueprint_algebra_fields_plonk_non_native_lookup_logic_ops_test,
-            blueprint_algebra_fields_plonk_non_native_comparison_checked_test,
-            blueprint_algebra_fields_plonk_non_native_comparison_unchecked_test,
-            blueprint_algebra_fields_plonk_non_native_comparison_flag_test,
-            blueprint_algebra_fields_plonk_non_native_equality_flag_test,
-            blueprint_algebra_fields_plonk_non_native_division_remainder_test,
-#            blueprint_non_native_plonk_scalar_non_native_range_test, TODO: enable once fixed.
-            blueprint_non_native_plonk_bool_scalar_multiplication_test,
-#            blueprint_non_native_plonk_add_mul_zkllvm_compatible_test, TODO: enable once fixed.
-            blueprint_hashes_plonk_decomposition_test,
-            blueprint_verifiers_placeholder_fri_cosets_test,
-            blueprint_hashes_plonk_sha256_process_test,
-            blueprint_hashes_plonk_sha512_process_test,
-            blueprint_hashes_plonk_sha256_test,
-            blueprint_hashes_plonk_sha512_test,
-            blueprint_algebra_fields_plonk_sqrt_test,
-            blueprint_verifiers_placeholder_fri_lin_inter_test,
-            blueprint_verifiers_placeholder_fri_array_swap_test,
-            blueprint_manifest_test,
-            blueprint_detail_huang_lu_test,
-            blueprint_private_input_test,
-            blueprint_verifiers_placeholder_permutation_argument_verifier_test,
-            blueprint_verifiers_placeholder_gate_argument_verifier_test,
-            blueprint_verifiers_placeholder_lookup_argument_verifier_test,
-            blueprint_verifiers_placeholder_f1_loop_test,
-            blueprint_verifiers_placeholder_f3_loop_test,
-            blueprint_verifiers_placeholder_gate_component_test,
-            blueprint_verifiers_flexible_pow_factor_test,
-            blueprint_proxy_test,
-#            blueprint_mock_mocked_components_test, TODO: Enable after code and test re-written.
-            blueprint_component_batch_test,
-            blueprint_verifiers_placeholder_expression_evaluation_component_test,
-            blueprint_verifiers_placeholder_final_polynomial_check_test,
-            blueprint_verifiers_flexible_swap_test,
-            blueprint_verifiers_flexible_additions_test,
-            blueprint_verifiers_flexible_multiplications_test,
-            blueprint_verifiers_flexible_poseidon_test,
-            blueprint_verifiers_flexible_constant_pow_test,
-            blueprint_verifiers_placeholder_verifier_test
-          ]  # Tests to execute
-        include: # Abused to enable proof generation for some tests; add more as needed

```

Comment:
Here some of the tests are run with additional arguments (i.e. `-DBLUEPRINT_PLACEHOLDER_PROOF_GEN=True`) and it doesn't seem that the behavior is replicated in the new tests.

Reply:
What about moving it from CI to CMakelists? I think, it's more clean.

Reply:
Done now